App/TopTrumps/TopTrumps.WebApp/Pages/Users.razor

```diff
@@ -0,0 +1,21 @@
+﻿@page "/users"
+@using TopTrumps.Data.DTOs
+@using TopTrumps.Data.Services
+@using TopTrumps.WebApp.Components
+
+
+@inject IUsersService _usersService
+
+@attribute [Authorize(Roles = "Admin")]
+
+<h3>Users</h3>
+
+@code {
+
+    private IEnumerable<User> _Users = new List<User>();
+
+    protected override async Task OnInitializedAsync()
+    {
+        _usersService = await IUsersService.GetAllUsers();
```

Comment:
This line is giving you an error. 

Here's a clue:
- `IUserService` is a type.
- `_usersService` is an object of that type. 
- You need an object to call methods on (see the text of the error).
![image](https://user-images.githubusercontent.com/2738442/139601675-5deedc94-340f-4995-be16-09b8dd75f70d.png)